language-support/java/bindings/src/test/java/com/daml/ledger/javaapi/data/codegen/json/JsonLfReaderTest.java

```diff
@@ -84,8 +83,9 @@ void testTimestamp() throws IOException {
         eq(
             "\"9999-12-31T23:59:59.999999Z\"",
             timestampUTC(9999, Month.DECEMBER, 31, 23, 59, 59, 999999)),
-        //            eq("\"1990-11-09T04:30:23.1234569Z\"", timestampUTC(1990, Month.NOVEMBER,
-        // 9,  4, 30, 23, 123457)), // TODO(raphael-speyer-da): Appropriate rounding
+        eq(
```

Comment:
re my comment above: some tests for out of bound numerics would probably be a good idea.

Reply:
Yes, makes sense. I'll do another PR with that.